test/capi/capiSavers.cpp

```diff
@@ -69,8 +69,6 @@ TEST_CASE("Save a paint into a file", "[capiSaver]")
     REQUIRE(tvg_saver_save(saver, paint3, TEST_DIR"/test.tvg", false) == TVG_RESULT_INSUFFICIENT_CONDITION);
 
     REQUIRE(tvg_saver_del(saver) == TVG_RESULT_SUCCESS);
-
-    tvg_paint_del(paint_empty);
```

Comment:
after #1004 this is not needed (in the first version of this pr I also included the changes that are present in 1005, since I missed it)

Reply:
Could you please remove this change in this PR? the patch was reverted already. Others looks good to me.